weighted_levenshtein/.gitignore

```diff
@@ -0,0 +1 @@
+clev.c
```

Comment:
nit: Add newline to the EOF

Reply:
added, not shown on github, but says 2 lines